docs/azure/index.yml

```diff
@@ -36,7 +36,7 @@ highlightedContent:
       title: Secure custom APIs with Microsoft Identity
       url: /learn/modules/identity-secure-custom-api/
     - itemType: get-started
-      title: Azure SDK for .NET
+      title: Azure .NET SDK
```

Comment:
It's spelled out Azure SDK for .NET in lots of places, including their GitHub repository. https://github.com/Azure/azure-sdk-for-net#azure-sdk-for-net

probably should remain as is?

Reply:
See #24428